packs/ibmmq/config.yml

```diff
@@ -0,0 +1,36 @@
+id: 924fd4b3-a6d1-4a6e-9e2c-b598f197f713
+
+# Name of the pack (required)
+name: ibmmq
+title: New Relic IBM MQ Monitor
+description: |
+ IBM MQ MQ allows independent and potentially non-concurrent applications on a distributed system to securely communicate with each other, using messages. MQ is available on a large number of platforms (both IBM and non-IBM), including z/OS (mainframe), IBM i, Transaction Processing Facility, UNIX (AIX, HP-UX, Solaris), HP NonStop, OpenVMS, Linux, and Microsoft Windows.
+
+ The New Relic IBM MQ Monitor allows you to monitor the performance of MQ Objects like channels and queues . The metrics collected include depth of your queues, how much space your queues have for additional messages, and the age of a queue’s oldest message. Analyzing metrics over time allows you to scale your instances accordingly in order to prevent full queues, which can delay message delivery. 
+
+summary: |
+  The New Relic IBM MQ Monitor enables performance monitoring of IBM MQ Objects like channels, topics and queues.
+
+icon: icon.png
+logo: logo.png
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+website: https://www.ibm.com/products/mq
+
+keywords:
+  - nrlabs
+  - nrlabs-data
+  - ibm mq
+  - mq
+
+# Authors of the pack (required)
+authors:
+  - New Relic Labs
+
+documentation:
+  - name: New Relic integration for IBM MQ 
+    description: |
+      The New Relic IBM MQ Monitor enables performance monitoring of IBM MQ Objects like channels, topics and queues.
+    url: https://github.com/newrelic-experimental/newrelic-ibm-mq-monitor
```

Comment:
```suggestion
    url: >-
    https://github.com/newrelic-experimental/newrelic-ibm-mq-monitor
```

Reply:
see this section of the contributor guide about this change

https://github.com/newrelic/newrelic-observability-packs/blob/main/CONTRIBUTING.md#documentation

Reply:
you will also need to tab the URL 1 tab.

Reply:
I quickly made this update. Most of the existing quickstarts don't follow this url format, so we should run a script to update them